grease/src/Grease/Macaw/Memory.hs

```diff
@@ -47,13 +47,12 @@ loadString ::
   , Mem.HasLLVMAnn sym
   , ?memOpts :: Mem.MemOptions
   ) =>
+  MM.AddrWidthRepr (MC.ArchAddrWidth arch) ->
```

Comment:
Why do we need to pass this in explicitly? Shouldn't we be able to just call `let addrWidth = MC.addrWidthRepr ?ptrWidth`, as was done before?